ProcessLib/RichardsFlow/RichardsFlowFEM.h

```diff
@@ -121,10 +120,11 @@ class LocalAssemblerData : public RichardsFlowLocalAssemblerInterface
                 dSwdPc = interpolated_Pc.getDerivative(
                     interpolated_Pc.getSupportMin());
 
+            double const Sw = interpolated_Pc.getValue(Pc);
             _saturation[ip] = Sw;
 
-            double k_rel = interpolated_Kr.getValue(Sw);
-            double drhow_dp(0.0);
+            double const k_rel = interpolated_Kr.getValue(Sw);
+            double const drhow_dp(0.0);
```

Comment:
Multiplying by 'drhow_dp`, which is zero and const,  make 'mass_mass_coeff` 0. Is this intended?